src/main/java/appeng/util/Lazy.java

```diff
@@ -0,0 +1,27 @@
+
+package appeng.util;
+
+
+import java.util.function.Supplier;
+
+
+public class Lazy<T> implements Supplier<T>
+{
+	final private Supplier<T> supplier;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AModifiersOrderCheck)